lib/tasks/export.rake

```diff
@@ -2,10 +2,6 @@ require "csv"
 require "fileutils"
 
 namespace :export do
-  def routes_helper
-    @routes_helper ||= Whitehall.url_maker
```

Comment:
I wonder why this helper was created in the first place. Memoising the url_maker suggests that creating it is an expensive operation - could you memoise it as a variable after `task document_list: :environment do` instead? I appreciate this means duplicating it for each rake task.

Reply:
Normally I'd agree: memoisation is an important technique for a running web app, where response time matters. I don't think it matters for a rake task, though; someone can always come an optimise it if it's really a problem.